homeassistant/components/ambient_station/sensor.py

```diff
@@ -337,162 +357,189 @@
         name="Soil Temp 10",
         native_unit_of_measurement=TEMP_FAHRENHEIT,
         device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_SOILTEMP1F,
         name="Soil Temp 1",
         native_unit_of_measurement=TEMP_FAHRENHEIT,
         device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_SOILTEMP2F,
         name="Soil Temp 2",
         native_unit_of_measurement=TEMP_FAHRENHEIT,
         device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_SOILTEMP3F,
         name="Soil Temp 3",
         native_unit_of_measurement=TEMP_FAHRENHEIT,
         device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_SOILTEMP4F,
         name="Soil Temp 4",
         native_unit_of_measurement=TEMP_FAHRENHEIT,
         device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_SOILTEMP5F,
         name="Soil Temp 5",
         native_unit_of_measurement=TEMP_FAHRENHEIT,
         device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_SOILTEMP6F,
         name="Soil Temp 6",
         native_unit_of_measurement=TEMP_FAHRENHEIT,
         device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_SOILTEMP7F,
         name="Soil Temp 7",
         native_unit_of_measurement=TEMP_FAHRENHEIT,
         device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_SOILTEMP8F,
         name="Soil Temp 8",
         native_unit_of_measurement=TEMP_FAHRENHEIT,
         device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     SensorEntityDescription(
         key=TYPE_SOILTEMP9F,
         name="Soil Temp 9",
```

Comment:
What are all these ? And why are they all enabled by default ?

Reply:
They are soil temperature sensors. Several models of weather station come with them – it's not automatic that every installation will have them, but it's not automatic they won't have them, either.

Reply:
Do we still add them if they are not there ?

Reply:
@balloob Negative, we only add entities for data we have: https://github.com/home-assistant/core/blob/f7ef973c680c2b19942e11b105cec10658465843/homeassistant/components/ambient_station/sensor.py#L565